templates/js/ai-bot/README.md.tpl

```diff
@@ -18,7 +18,9 @@ The app template is built using the Teams AI library, which provides the capabil
 > To run the AI Chat Bot template in your local dev machine, you will need:
 >
 > - [Node.js](https://nodejs.org/), supported versions: 16, 18
+{{#enableTestToolByDefault}}
```

Comment:
Should it be {{^enableTestToolByDefault}}?

Reply:
good catch, fixed